runtime/packages/core-modules/assembly/system.ts

```diff
@@ -46,6 +46,9 @@ export class System {
      * @param header Header instance
     */
     static initialize(header: Header): void{
+        // maximum number of blocks
```

Comment:
- I think that it would be better if we clarify that this is the blocks to store before pruning them.
- I think that it would be better to have a `static readonly ` property with the number f.e:
```JavaScript
// specified the number of blocks to keep before pruning them
static readonly NUMBER_OF_BLOCKS_TO_KEEP int32 = 1000 
```
and once we need to set it on storage using the `BHSH_COUNT` we can create new `Uint32` instance.